File: Java/Easy - Basic Programs/shape.java
/*
 * Author : musaraf
 * Email : nowsathmusaraf037@gmail.com
 * Date : 09/07/2024
 * Description : user what the shape area or circumference
 */

 import java.util.Scanner;
public class shape {

    public static void rectangle(){

        Scanner sc = new Scanner(System.in);

        System.out.print("Enter the length of the rectangle : ");
        double length = sc.nextDouble();
        System.out.print("Enter the width of the rectangle : ");
        double width = sc.nextDouble();
        double area = length * width;
        double circumference = 2 * (length + width);

        System.out.println("Rectangle Area is : "+area);
        System.out.println("Rectangle Circumference is : "+circumference);

        sc.close();
    }
    public static void circle(){

        Scanner sc = new Scanner(System.in);

        final double PI = 3.14;
        System.out.print("Enter the radius of the circle : ");
        double radius = sc.nextDouble();
        double area = PI * radius * radius;
        double circumference = 2 * PI * radius;

        System.out.println("Circle Area is : "+area);
        System.out.println("Circle Circumference is : "+circumference);

        sc.close();
    }
    public static void square(){

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter the length of the rectangle");
        double length = sc.nextDouble();
        double area = length * length;
        double circumference = 4 * length;

        System.out.println("Square Area is : "+area);
        System.out.println("Square Circumference is : "+circumference);

        sc.close();
    }
    public static void triangle(){

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the length of the triangle");
        double length = sc.nextDouble();
        System.out.println("Enter the width of the triangle");
        double width = sc.nextDouble();
        
        double area = length * width / 2;
        double circumference = length + width + Math.sqrt(length * length + width * width);

        System.out.println("Triangle Are is : "+area);
        System.out.println("Triangle Circumference is : "+circumference);

        sc.close();
    }
    public static void main(String [] args){

        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the shape NO");
        System.out.println("1. Rectangle");
        System.out.println("2. circle");
        System.out.println("3. square");
        System.out.println("4. Triangle");

        int shape = sc.nextInt();

        switch (shape) {
            case 1:
                rectangle();
                break;
            case 2:
                circle();
                break;
            case 3:
                square();
                break;
            case 4:
                triangle();
                break;

            default:
            System.out.println("Invalid Shape NO");
        }
        sc.close();
    }
}

File: C#/Medium/shape.cs
/*
* Author : musaraf
* Email : nowsathmusaraf037@gmail.com
* Date : 11/07/2024
* Description : choice a shape calculate area & circumference
*/

namespace shape{
    class program{
        void rectangle(){


        Console.WriteLine("Enter the length of the rectangle : ");
        double length = Convert.ToDouble(Console.ReadLine());
        Console.WriteLine("Enter the width of the rectangle : ");
        double width = Convert.ToDouble(Console.ReadLine());
        double area = length * width;
        double circumference = 2 * (length + width);

        Console.WriteLine("Rectangle Area is : "+area);
        Console.WriteLine("Rectangle Circumference is : "+circumference);

    }
    void circle(){

        const double PI = 3.14;
        Console.WriteLine("Enter the radius of the circle : ");
        double radius = Convert.ToDouble(Console.ReadLine());
        double area = PI * radius * radius;
        double circumference = 2 * PI * radius;

        Console.WriteLine("Circle Area is : "+area);
        Console.WriteLine("Circle Circumference is : "+circumference);

    }
    void square(){


        Console.WriteLine("Enter the length of the rectangle");
        double length = Convert.ToDouble(Console.ReadLine());
        double area = length * length;
        double circumference = 4 * length;

        Console.WriteLine("Square Area is : "+area);
        Console.WriteLine("Square Circumference is : "+circumference);

    }
    void triangle(){

        Console.WriteLine("Enter the length of the triangle");
        double length = Convert.ToDouble(Console.ReadLine());
        Console.WriteLine("Enter the width of the triangle");
        double width = Convert.ToDouble(Console.ReadLine());
        
        double area = length * width / 2;
        double circumference = length + width + Math.sqrt(length * length + width * width);

        Console.WriteLine("Triangle Are is : "+area);
        Console.WriteLine("Triangle Circumference is : "+circumference);

    }
    public static void main(String [] args){

        Console.WriteLine("Enter the shape NO");
        Console.WriteLine("1. Rectangle");
        Console.WriteLine("2. circle");
        Console.WriteLine("3. square");
        Console.WriteLine("4. Triangle");

        int shape = Convert.ToInt32(Console.ReadLine());

        switch (shape) {
            case 1:
                rectangle();
                break;
            case 2:
                circle();
                break;
            case 3:
                square();
                break;
            case 4:
                triangle();
                break;

            default:
            Console.WriteLine("Invalid Shape NO");
        }
    }
    }
}